### DISCUSSION TITLE
gistic2脚本 安装失败怎么解决

### DISCUSSION THREAD

**Discussion by sonoko97** [ShixiangWang/self-study#75] [Q&A] [answered]
使用gistic2安装脚本时最后一步显示没有JRE，但是我已经安装了，不知道该如何解决
<img width="817" alt="QQ20210421-213317@2x" src="https://user-images.githubusercontent.com/82953018/115563056-b43e6600-a2e9-11eb-92be-1061a8aa8cff.png">

**Comment by ShixiangWang** [comment, marked as answer]
gistic2 好像没法在 MacOS 下工作。建议使用 docker 或者我在 hiplot 上提供的插件 https://hiplot.com.cn/advance/gistic2

**Comment by sonoko97** [comment]
谢谢回复啦！！hiplot上的插件我用了，下载下来一个maf文件后就不知道怎么分析了(._.)跟我学习的教程不一样了（😭）------------------&nbsp;原始邮件&nbsp;------------------
***@***.***&gt;
发送时间:&nbsp;2021年4月21日(星期三) 晚上11:07
***@***.***&gt;;
***@***.******@***.***&gt;;
主题:&nbsp;Re: [ShixiangWang/self-study] gistic2脚本 安装失败怎么解决 (#75)

**Reply by ShixiangWang** [reply]
不需要maf文件，用seg文件

**Reply by ShixiangWang** [reply]
![image](https://user-images.githubusercontent.com/25057508/115583391-6e3ecd80-a2fc-11eb-8dc2-b79c213c0581.png)

默认zhi只要传一个文件就够了。

**Comment by sonoko97** [comment]
不是不是，seg我传好了，下载下来这些文件下一步该怎么处理呢？怎么没有txt文件。。还有为啥这个插件不需要上传Markers File啊
谢谢你为我解答！

<img width="570" alt="hiplot" src="https://user-images.githubusercontent.com/82953018/115650695-4ed78d00-a35c-11eb-87c2-7643010cde30.png">




------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "ShixiangWang/self-study"                                                                                    ***@***.***&gt;;
发送时间:&nbsp;2021年4月21日(星期三) 晚上11:51
***@***.***&gt;;
***@***.******@***.***&gt;;
主题:&nbsp;Re: [ShixiangWang/self-study] gistic2脚本 安装失败怎么解决 (#75)





 

 
默认zhi只要传一个文件就够了。

**Reply by ShixiangWang** [reply]
很简单，因为这些文件是通用的，已经放在服务器上了。如果自己安装和使用当然要下载。
